pyked/converters.py

```diff
@@ -0,0 +1,741 @@
+
+    for dp in c.datapoints:
+        datapoint = etree.SubElement(datagroup, 'dataPoint')
+        for idx in property_idx:
+            value = etree.SubElement(datapoint, idx)
+            value.text = str(getattr(dp, property_idx[idx].replace(' ', '_')).magnitude)
+
+    # if RCM and has volume history, need a second dataGroup
+    if (len(c.datapoints) > 1 and
+        any([getattr(dp, 'volume_history', None) for dp in c.datapoints])
+        ):
+        raise NotImplementedError('Error: ReSpecTh files do not support multiple datapoints '
+                                  'with a volume history.'
+                                  )
+        # TODO: what if they share the same history? Does this happen?
+    elif getattr(c.datapoints[0], 'volume_history', None):
+        datagroup = etree.SubElement(root, 'dataGroup')
+        datagroup.set('id', 'dg1')
+        datagroup_link = etree.SubElement(datagroup, 'dataGroupLink')
+        datagroup_link.set('dataGroupID', '')
+        datagroup_link.set('dataPointID', '')
+
+        # Volume history has two properties: time and volume.
+        volume_history = c.datapoints[0].volume_history
+        prop = etree.SubElement(datagroup, 'property')
+        prop.set('description', '')
+        prop.set('name', 'time')
+        prop.set('units', str(volume_history.time.units))
+        time_idx = 'x{}'.format(len(property_idx) + 1)
+        prop.set('id', time_idx)
+        prop.set('label', 't')
+
+        prop = etree.SubElement(datagroup, 'property')
+        prop.set('description', '')
+        prop.set('name', 'volume')
+        prop.set('units', str(volume_history.volume.units))
+        volume_idx = 'x{}'.format(len(property_idx) + 2)
+        prop.set('id', volume_idx)
+        prop.set('label', 'V')
+
+        for time, volume in zip(volume_history.time, volume_history.volume):
+            datapoint = etree.SubElement(datagroup, 'dataPoint')
+            value = etree.SubElement(datapoint, time_idx)
+            value.text = str(time.magnitude)
+            value = etree.SubElement(datapoint, volume_idx)
+            value.text = str(volume.magnitude)
+
+    # In ReSpecTh files all datapoints share ignition type
+    ignition = etree.SubElement(root, 'ignitionType')
+    if c.datapoints[0].ignition_type['target'] == 'pressure':
+        ignition.set('target', 'P')
+    elif c.datapoints[0].ignition_type['target'] == 'temperature':
+        ignition.set('target', 'T')
+    else:
+        # options left are species
+        ignition.set('target', c.datapoints[0].ignition_type['target'])
+    ignition.set('type', c.datapoints[0].ignition_type['type'])
+
+    et = etree.ElementTree(root)
+    filename_out = os.path.join(
+        output_path,
+        os.path.splitext(os.path.basename(filename_ck))[0] + '.xml'
+        )
+    et.write(filename_out, pretty_print=True, encoding='utf-8', xml_declaration=True)
+
+    return filename_out
+
+
+if __name__ == '__main__':
```

Comment:
Can you add this to a function called `main` or something, so that we can create entry points? See [ck2cti](https://github.com/Cantera/cantera/blob/master/interfaces/cython/cantera/ck2cti.py#L2297) for an example